workers/vita/Dockerfile

```diff
@@ -17,56 +17,77 @@ RUN sed 's/^deb \(.*\)/deb-src \1/' /etc/apt/sources.list \
 ENV VITASDK=/opt/vitasdk
 ENV PATH=$PATH:$VITASDK/bin
 
-ARG URL=https://github.com/vitasdk/autobuilds/releases/download/master-linux-v567/vitasdk-x86_64-linux-gnu-2017-10-29_09-48-44.tar.bz2
+#Install VitaSDK base
+ARG URL=https://github.com/vitasdk/autobuilds/releases/download/master-linux-v611/vitasdk-x86_64-linux-gnu-2017-12-01_14-01-36.tar.bz2
 
 RUN wget --progress=bar:force:noscroll -O - $URL \
 	| tar --no-same-owner --no-same-permissions -C /opt/ -xjf -
 
-# These insecure, unversioned downloads are not great, but OpenSSL and SDL2 are
-# not trivially buildable for this target so we use these packages for the
-# moment. At the time of the first build these corresponded to
-# https://github.com/vitasdk/packages/commit/b62cb68c349f8f389a4b4e57356c398c43f5531d
-# in case someone needs to rebuild them from source later
-RUN wget --progress=bar:force:noscroll -O - http://dl.vitasdk.org/openssl.tar.xz \
-	| tar --no-same-owner --no-same-permissions -C $VITASDK/arm-vita-eabi -xJf -
-RUN wget --progress=bar:force:noscroll -O - http://dl.vitasdk.org/curl.tar.xz \
-	| tar --no-same-owner --no-same-permissions -C $VITASDK/arm-vita-eabi -xJf -
-RUN wget --progress=bar:force:noscroll -O - http://dl.vitasdk.org/sdl2.tar.xz \
-	| tar --no-same-owner --no-same-permissions -C $VITASDK/arm-vita-eabi -xJf -
-RUN wget --progress=bar:force:noscroll -O - http://dl.vitasdk.org/sdl_net.tar.xz \
-	| tar --no-same-owner --no-same-permissions -C $VITASDK/arm-vita-eabi -xJf -
-
+#Install git
+RUN apt-get -y install git
+
+#Install sudo
+RUN apt-get -y install sudo
+
+#Install curl
+RUN apt-get -y install curl
+
+#Install Vita versions of most common libraries, including ogg, png, zlib etc.
+RUN git clone https://github.com/vitasdk/vdpm && \
+	cd vdpm && \
+	./install-all.sh && \
+	cd ..
+
+#Install recent SDL2 port for Vita
+RUN git clone https://github.com/rsn8887/SDL-Vita && \
+	cd SDL-Vita && \
+	make -f Makefile.vita install && \
+	cd ..
+
+#Install libmad for Vita
+RUN git clone https://github.com/Cpasjuste/libmad-psp2 && \
+	cd libmad-psp2 && \
+	make install && \
+	cd ..
+
+#Enable freetype2 on Vita by adding a patched config file
+RUN git clone https://github.com/rsn8887/vita-freetype-config && \
+	cd vita-freetype-config && \
+	chmod 777 freetype-config && \
```

Comment:
This is a terribly wrong permission for any file on basically any *nix system. With this, the file is made writeable by any user on the system, which means any unprivileged user can replace its content with anything they want, which is doubly bad since it is also marked as executable. Furthermore, the permissions of files are stored in the repository, so since it is your repository, the problem should be fixed in the correct place and the files should be committed with the correct permissions, not ad-hoc fix them outside later on like this.